packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -118,5 +118,21 @@ export class CXLAppLayoutElement extends LitElement {
     registerGlobalStyles(cxlAppLayoutGlobalStyles, {
       moduleId: 'cxl-app-layout-global',
     });
+
+    this.focusAnchor();
+  }
+
+  focusAnchor() {
```

Comment:
Perhaps `maybeFocusAnchor()`?

Reply:
How about `shouldFocusAnchor`?

Reply:
> How about `shouldFocusAnchor`?

That reads like it would return a boolean for a state check.

Reply:
`focusAnchorIfAvailable`?

Reply:
I don't mind using `maybeFocusAnchor`

Reply:
> I don't mind using `maybeFocusAnchor`

I adopted it from WP.

There may have been some articles to caution against `maybe`-convention, but I forget what the arguments were.